frontend/src/Pages/PostDetails.module.css

```diff
@@ -0,0 +1,31 @@
+:root {
+  --bg-main: #606c38;
+  --bg-two: #283618;
+  --bg-three: #d3bfa9;
+  --bg-four: #dda15e;
+  --color-main: #f8f8ff;
+  --color-two: #f8f8ff;
+  --color-three: #444444;
+  --color-four: #444444;
+  --page-break: 820px;
+  --padding-mobile: 2%;
+  --padding-desktop-leftright: 4%;
+  --padding-desktop-above: 0.1%;
+}
```

Comment:
Bitte diese globalen Variablen auch global definiert lassen (also in der app.css) und hier entfernen, sonst haben wir bald keinen Überblick, wo diese Variablen überall definiert sind.

Warum willst du diese Variablen hier überhaupt noch mal definieren? Die Variablen kannst du ohne Probleme auch einfach so verwenden, ohne sie hier erneut zu definieren.

Will sagen: wenn du den Code hier entfernst wird alles genau so funktionieren wie mit diesem Code.